cli/deps.edn

```diff
@@ -32,6 +32,11 @@
                  :jvm-opts ["-Xmx2g" "-server"]}
            :debug {:extra-deps {nrepl/nrepl {:mvn/version "0.9.0"}
                                 cider/cider-nrepl {:mvn/version "0.28.3"}}}
+           :perf {:extra-deps {criterium/criterium {:mvn/version "0.4.6"}
```

Comment:
Maybe just rename to a better name consistency?
```suggestion
           :performance {:extra-deps {criterium/criterium {:mvn/version "0.4.6"}
```

Reply:
👍  done.